Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [FiguraMC/Figura](https://hosted.weblate.org/projects/figuramc/mod/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/figuramc/mod/horizontal-auto.svg)

Covkie commented:
BIG TRANSLATIONS ARE HERE :crab: :crab: :crab: 


Something to note: 5 hours after changes are done at https://hosted.weblate.org/projects/figuramc/mod/ they will be pushed to Github and a pull request will be made (if there is not already one).


~~Also can actions be disabled for prs from a specifc user?~~

Covkie committed:
Translated using Weblate (English (United States))

Currently translated at 100.0% (1594 of 1594 strings)

Translation: FiguraMC/Figura
Translate-URL: https://hosted.weblate.org/projects/figuramc/mod/en_US/
